wrench/src/yaml_helper.rs

```diff
@@ -145,6 +145,15 @@ impl YamlHelper for Yaml {
     }
 
     fn as_matrix4d(&self) -> Option<LayoutTransform> {
+        if let Some(mut nums) = self.as_vec_f32() {
```

Comment:
It's probably still possible to use map() to do this, but we can re-visit when the code gets updated.